tests/integration/cli/classification/test_classification.py

```diff
@@ -222,6 +224,15 @@ def test_pot_optimize(self, template, tmp_dir_path):
     def test_pot_eval(self, template, tmp_dir_path):
         pot_eval_testing(template, tmp_dir_path, otx_dir, args)
 
+    @e2e_pytest_component
+    @pytest.mark.skipif(TT_STABILITY_TESTS, reason="This is TT_STABILITY_TESTS")
+    @pytest.mark.skipif(MULTI_GPU_UNAVAILABLE, reason="The number of gpu is insufficient")
```

Comment:
How about `The number of gpu is scarce to run multi-gpu training.`

Reply:
I think this sentence also ok